start.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -eu
+
+repodir=/app/data
+search_locations="file:$repodir,file:$repodir/labels,file:$repodir/ldap,file:$repodir/lookups,file:$repodir/rules"
```

Comment:
How do the `labels`, `ldap`, `lookups`, and `rules` folders get created?  Also, the config server uses more folders: `branding`, `pdf-stylesheets` and `spring`.

Reply:
The whole `repodir` folder has to be shared with the ArkCase container, but I don't see any volume mounts or volumes or whatnot.  If the files in this MR are incomplete, the MR should be marked `WIP` ("work in progress").  Or else, the missing files should be added to the MR.

Reply:
.arkcase extention repo which is shared with arkcase core will be cloned at `ark_extension_path` location path. And value for `ark_extension_path` is given as a environment variable to the POD. For example if .arkcase repo is clone at `/data` the value of ark_extension_path will be given as `/data/.arkcase`

Reply:
@david-oc-miller @naveen-armedia  can this conversation be resolved?

Reply:
Yes @JonHolman